### PR TITLE
fix: add backwards compatibility export for DataTableEmptyState

### PR DESCRIPTION
**Pull request by jordankoschei-okta**
[REPLACE_WITH_JIRA_TICKET_NUMBER](https://oktainc.atlassian.net/browse/REPLACE_WITH_JIRA_TICKET_NUMBER)

## Testing & Screenshots

- [ ] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

File: packages/odyssey-react-mui/src/EmptyState.tsx [hunk 1]
```diff
@@ -76,3 +76,4 @@ const MemoizedEmptyState = memo(EmptyState);
 MemoizedEmptyState.displayName = "EmptyState";
 
 export { MemoizedEmptyState as EmptyState };
+export { MemoizedEmptyState as DataTableEmptyState };
```
Comment (bryancunningham-okta) [inline packages/odyssey-react-mui/src/EmptyState.tsx:79]: I'm not sure we want to change the name of this component do we?

Comment (bryancunningham-okta) [reply packages/odyssey-react-mui/src/EmptyState.tsx:79]: That would break other folks imports. Also breaks the import in our own `DataTable.tsx` file

**Review by KevinGhadyani-Okta — APPROVED** [review]
This is currently a bug, so this fixes it.

**Comment by bryancunningham-okta** [comment]
Didn't we intentionally make this component data table agnostic though? Can't we just create another export in the index file or something to fix the bug?

**Comment by bryancunningham-okta** [comment]
Never mind, I read this change as removing the other export and exporting it as ONLY DataTableEmptyState. Carry on then :)

**Review by bryancunningham-okta — APPROVED** [review]

**Pull request closed without merging**